### PR TITLE
Fix webapp resource lookup

### PR DESCRIPTION
**Pull request by daniel-beck**
Currently, Core webapp resources such as `/help/system-config/master-slave/usage.html` do not work. This was an oversight in the previous PR. This change should address the problem.

@LinuxSuRen Please confirm.

### CODE REVIEW

**Comment by daniel-beck** [comment]
The last commit fixes the URL like http://localhost:8080/plugin/credentials/help/domain/name.html

My apologies for getting this wrong, I have no idea how these slipped through.

File: pom.xml [hunk 1]
```diff
@@ -56,6 +56,10 @@
       <resource>
         <directory>plugins/credentials-plugin/src/main/resources</directory>
       </resource>
+      <resource>
+        <directory>plugins/credentials-plugin/src/main/webapp</directory>
+        <targetPath>webapp</targetPath>
```
Comment (daniel-beck) [inline pom.xml]: So the interesting bit is, as currently proposed, plugin `webapp/` resources would like in a subdirectory, while core `webapp/` resources would not. This is due to `#getPluginResource` being exclusively used for plugin webapp resources, while `#getResource` is used for other resources as well.

**Comment by daniel-beck** [comment]
Conflicted because the other PR was squash-merged, with a regular merge commit this would not have happened.

**Comment by LinuxSuRen** [comment]
It might because I made some changes at #77

**Comment by LinuxSuRen** [comment]
Let me fix this conflict.

**Comment by daniel-beck** [comment]
@LinuxSuRen 

> It might because I made some changes at #77

Git understand commit history and can easily merge without conflict — unless you use squash-merges or rebase-merges, as those will create different commits, and histories diverging so that the tool cannot automatically merge anymore.

**Comment by LinuxSuRen** [comment]
Yes, I understand this. I just want to keep a clean commit log.

**Pull request merged**